internal/logger/logger.go

```diff
@@ -46,13 +46,13 @@ var (
 // config.
 // Here, background true means, this InitLogFile has been called for the
 // background daemon.
-func InitLogFile(filename string, format string, level config.LogSeverity) error {
+func InitLogFile(logConfig config.LogConfig) error {
```

Comment:
Is this function unit-tested ? If yes, those should be updated, and I don't see them here.